content/docs/command-reference/update.md

```diff
@@ -49,6 +49,17 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
+- `--to-remote` - sync the changes directly on the remote storage, rather than
+  on the local <abbr>workspace</abbr>.
```

Comment:
Agree on Ivan's Q. For now (based on the published text in https://dvc.org/doc/command-reference/import-url#options here's a first copy edit (not necessarily final but we could commit it for now):

```suggestion
- `--to-remote` - update the external data, but don't move it into the
  workspace, nor cache it.
  [Transfer it](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  directly to remote storage (the default one, unless `-r` is specified)
  instead. Use `dvc pull` to get the data locally.
```

Reply:
`update the external data` sounds like we are updating the data itself. Can we say something like "update .dvc files and move new version of the external data into the remote storage ..."?

Reply:
Sure, maybe this then:

```suggestion
- `--to-remote` - update the import `.dvc` file and
  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  the new data directly to remote storage (the default one unless `-r` is used).
  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
  data locally.
```